drivers/usb/device/usb_dc_stm32.c

```diff
@@ -36,15 +36,27 @@ LOG_MODULE_REGISTER(usb_dc_stm32);
 #error "Only one interface should be enabled at a time, OTG FS or OTG HS"
 #endif
 
+/*
+ * Vbus sensing is determined based on the presence of the hardware detection pin(s) in the device
+ * tree. E.g: pinctrl-0 = <&usb_otg_fs_vbus_pa9 ...>;
+ *
+ * The detection pins are dependent on the enabled USB driver and the physical interface(s) offered
+ * by the hardware. These are mapped to PA9 and/or PB13 (subject to MCU), being the former the most
+ * widespread option.
+ */
```

Comment:
Please reformat this lines, look at the style in the file, that should not exceed the preferred length.

Reply:
AFAIK it is not required and usually do not accept line length changes caused by clang-format.